Makefile

```diff
@@ -122,11 +128,11 @@ getlog: $(AFSBOTCFG_LOGDIR)
 # Test targets
 #
 .PHONY: lint
-lint: $(PACKAGES)
+lint: $(PACKAGES) collections
 	$(INFO) "Running lint checks"
 	$(ACTIVATED) $(MAKE) -C src lint
 	$(ACTIVATED) yamllint $(YAML_FILES)
-	$(ACTIVATED) ansible-lint $(LINT_OPTIONS)
+	$(ACTIVATED) ANSIBLE_COLLECTIONS_PATH="${LINT_ANSIBLE_COLLECTIONS_PATH}" ansible-lint $(LINT_OPTIONS)
```

Comment:
Instead of doing this before lint, I think we want to specify the collection path as part of the ACTIVATED macro, so it is the same for ansible-playbook, ansible-lint, and molecule.

For example:

index 5e27777..4df463f 100644
--- a/Makefile
+++ b/Makefile
@@ -65,11 +65,12 @@ VAULT_KEYFILE=.vault-afsbotcfg
 PACKAGES=.packages
 ifdef VIRTUAL_ENV
 PIP=$(VIRTUAL_ENV)/bin/pip
+ACTIVATED=ANSIBLE_COLLECTIONS_PATH=$(CURDIR)/collections
 else
 VENV=.venv
 PIP=$(VENV)/bin/pip
 ACTIVATE=$(VENV)/bin/activate
-ACTIVATED=. $(ACTIVATE);
+ACTIVATED=. $(ACTIVATE); ANSIBLE_COLLECTIONS_PATH=$(CURDIR)/collections
 endif